doc/dev/cephadm/host-maintenance.rst

```diff
@@ -0,0 +1,100 @@
+================
+Host Maintenance
+================
+
+All hosts that support Ceph daemons need to support maintenance activity, whether the host
+is physical or virtual(vm or cloud). This means that management workflows should provide
```

Comment:
Should be a space before `(vm or cloud)`, but really that text should just be removed, it doesn't add any meaning to the sentence.